build-system/server/new-server/transforms/stories/stories-transform.ts

```diff
@@ -36,6 +36,8 @@ function sidegradeStories(script: PostHTML.Node): PostHTML.Node {
 /**
  * Replace the src for every stories script tag.
  */
-export default function(tree: PostHTML.Node): void {
-  tree.match({tag: 'script'}, sidegradeStories);
+export default function(options: any): (tree: PostHTML.Node) => void {
```

Comment:
lets not use `any`, lets define an interface with a strict set of fields. lets take advantage of types

Reply:
Done.
I added them to every transformer, but right now the interface is empty for the other transformers.